.github/report-release-vulnerabilities.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# download the release.yaml
+gh release download --clobber --pattern release.yaml --output /tmp/release.yaml
+
+# extract the images
+readarray -t images < <(grep ghcr.io /tmp/release.yaml | sed -E 's/(image|value)://' | tr -d ' ' | sort -u)
+
+# capture whether vulnerabilities exist
+hasVulnerabilities=false
+
+# iterate the images
+true>/tmp/report.md
+for image in "${images[@]}"; do
+  echo "[INFO] Checking image ${image}"
+  echo "## ${image}" >>/tmp/report.md
+
+  # OS vulnerabilities
+  echo "  [INFO] Checking for OS vulnerabilities"
+  echo "### OS vulnerabilities" >>/tmp/report.md
+  osVulns="$(trivy image --ignore-unfixed --no-progress --pkg-types os --scanners vuln --timeout 10m "${image}" --format json)"
```

Comment:
With regards to to its recent tendency to be subject to 429ish problems, should we think about safeguards or mitigations here? We could use the GHCR registry for its database, if this isn't the default, or the username/password like in the other PR?